e2e_tests/support/global.d.ts

```diff
@@ -0,0 +1,108 @@
+/// <reference types="cypress" />
+
+export {}
```

Comment:
Where did this file come from? Is there no @types/cypress?

Reply:
Cypress ships with type definitions, these type definitions I wrote here are merged with the standard ones and are just to have definitions for the custom commands and config variables we add

Reply:
Ah, nice!